packages/vscodium-deb/vscodium-deb.pacscript

```diff
@@ -1,7 +1,7 @@
 name="vscodium-deb"
 gives="codium"
-version="1.60.0"
-url="https://github.com/VSCodium/vscodium/releases/download/$version/codium_${version}-1630974030_amd64.deb"
+version="1.60.2"
+url="https://github.com/VSCodium/vscodium/releases/download/1.60.2/codium_1.60.2-1632356529_amd64.deb"
```

Comment:
Why is the `url` no longer integrated with the `version`?

Reply:
I didn't find it very "useful". I still need to touch the url to change the randomly generated numbers (1632356529) by the GitHub Actions every time there is an update.

Reply:
Okay then.